src/adc.rs

```diff
@@ -676,15 +676,24 @@ adc_pins! {
     Channel9: (gpiob::PB1<Analog>, 9u8),
 }
 
-#[cfg(all(feature = "stm32l052", any(feature = "lqfp64", feature = "tfbga64",),))]
+#[cfg(all(
+    all(
+        any(feature = "io-STM32L051", feature = "io-STM32L071"),
+        any(feature = "stm32l0x2", feature = "stm32l0x3"),
```

Comment:
Do we still need these two features, or can we remove them from the cfg filter? (In other words, does `io-*` handle everything already?)